fr.free.Qccrypt.yml

```diff
@@ -0,0 +1,34 @@
+app-id: fr.free.Qccrypt
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: qccrypt
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=host
+  - --device=dri
+modules:
+  - name: qccrypt
+    buildsystem: qmake
+    sources:
+      - type: archive
+        url: http://qccrypt.free.fr/download/qccrypt-0.9.1-src.zip
+        sha256: 58b56e7dcf2219c08044d7e9ae1d41144452d178bda30040c707e4dcd290e081
+      - type: file
+        path: fr.free.Qccrypt.desktop
+      - type: file
+        path: qccrypt-256-x256.png
```

Comment:
this is in `qccrypt-0.9.1-src/ressources/images/qccrypt.png`. So it has no need to be in this repository.

Reply:
Humm, ok I didn't know that I could use an image from the source code. I will try to make it work.